samples/templates/default-azuredeploy-docker.json

```diff
@@ -123,6 +123,13 @@
                 "description": "Use Cosmos DB free tier."
             }
         },
+        "cosmosDbByokKeyUrl": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "Key url for the Cosmos DB customer managed key. If not provided a system managed key will be used."
```

Comment:
Should we add a note saying an invalid setting will cause the service not to start?